dart/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gitpod/workspace-full
+
+
+ENV PATH=/usr/lib/dart/bin:$PATH
+
+USER root
+
+RUN curl https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - && \
+    curl https://storage.googleapis.com/download.dartlang.org/linux/debian/dart_stable.list > /etc/apt/sources.list.d/dart_stable.list && \
+    apt-get update && \
+    apt-get -y install  build-essential dart libkrb5-dev gcc make && \
+    apt-get clean && \
+    apt-get -y autoremove && \
+    apt-get -y clean && \
+    rm -rf /var/lib/apt/lists/*;
```

Comment:
Nit: I think this Dockerfile is already included in `flutter/Dockerfile`. I don't think we'll want to officially support a dedicated `gitpod/workspace-dart` environment image, but maybe we could clearly highlight the Dart-specific lines in the `flutter/Dockerfile` e.g. with comments like `# Install Dart` and `# Install Flutter`? That way if some projects do really need a Dart-only environment one day, they can simply copy the Dart lines into a custom Dockerfile for their project.

Reply:
The reason I specifically added a seperate no-VNC Dart stack is because there is support beyond Flutter (Angel HTTP Framework, Flutter GL wrapper, etc.) for Dart, and it would make sense to actually support the baseline language that Flutter relies on, primarily because of this reasons.